src/entities/course/types.ts

```diff
@@ -17,3 +17,8 @@ export type Course = {
 };
 
 export type CourseStatus = 'planned' | 'available' | 'upcoming';
+
+export type CourseItemData = Pick<Course, 'title' | 'language' | 'startDate' | 'detailsUrl'> & {
```

Comment:
```suggestion
export type CourseItemData = Pick<Course, 'title' | 'language' | 'startDate' | 'detailsUrl'>
  & {
```

Reply:
30.09.24 on the meeting decide to leave it as it is